engine/src/test/java/io/camunda/zeebe/engine/util/client/ProcessInstanceClient.java

```diff
@@ -67,6 +68,12 @@ public ProcessInstanceCreationClient withVariable(final String key, final Object
       return this;
     }
 
+    public ProcessInstanceCreationClient withStartInstruction(
+        final ProcessInstanceCreationStartInstruction instruction) {
+      processInstanceCreationRecord.addStartInstruction(instruction);
+      return this;
+    }
```

Comment:
I don't agree with this. The record we are creating contains a `startInstruction` property. It makes perfect sense to me to have a `withStartInstruction` method which will add this instruction. Whether it is start before or after an element is irrelevant. Especially since there is only one possibility. If we ever decide to support multiple options they should be added to the start instruction that is passed.

I don't like to specify it in the test names either. We don't know the concept of starting before or after an element in Zeebe, so why add it to the test name? I could understand doing it in the Java client for backwards compatibility down the road, but here I feel it's unnecessary.

Reply:
Okay :smile: I don't force you to change.